distributions/aws/test/e2e/tests/test_rds_s3.py

```diff
@@ -0,0 +1,444 @@
+        print(metadata.params)  # These needed to be logged
+        print("Created metadata file for TestRDSS3", metadata_file)
+
+    def test_kfp_experiment(self, setup, kfp_client, cfn_stack, cfn_client):
+        stack_outputs = get_stack_outputs(cfn_client, cfn_stack["stack_name"])
+
+        name = rand_name("experiment-")
+        description = rand_name("description-")
+        experiment = kfp_client.create_experiment(
+            name, description=description, namespace=DEFAULT_USER_NAMESPACE
+        )
+
+        assert name == experiment.name
+        assert description == experiment.description
+        assert DEFAULT_USER_NAMESPACE == experiment.resource_references[0].key.id
+
+        mysql_client = create_mysql_client(
+            user=cfn_stack["params"]["DBUsername"],
+            password=cfn_stack["params"]["DBPassword"],
+            host=stack_outputs["RDSEndpoint"],
+            database="mlpipeline",
+        )
+
+        resp = mysql.query(
+            mysql_client, f"select * from experiments where Name='{name}'"
+        )
+        assert len(resp) == 1
+        assert resp[0]["Name"] == experiment.name
+        assert resp[0]["Description"] == experiment.description
+        assert resp[0]["Namespace"] == experiment.resource_references[0].key.id
+
+        resp = kfp_client.get_experiment(
+            experiment_id=experiment.id, namespace=DEFAULT_USER_NAMESPACE
+        )
+
+        assert name == resp.name
+        assert description == resp.description
+        assert DEFAULT_USER_NAMESPACE == resp.resource_references[0].key.id
+
+        kfp_client.delete_experiment(experiment.id)
+
+        resp = mysql.query(
+            mysql_client, f"select * from experiments where Name='{name}'"
+        )
+        assert len(resp) == 0
+
+        try:
+            kfp_client.get_experiment(
+                experiment_id=experiment.id, namespace=DEFAULT_USER_NAMESPACE
+            )
+            raise AssertionError("Expected KFPApiException Not Found")
+        except KFPApiException as e:
+            assert "Not Found" == e.reason
+
+        mysql_client.close()
+
+    def test_run_pipeline(self, setup, kfp_client, cfn_stack, cfn_client, s3_client):
+        stack_outputs = get_stack_outputs(cfn_client, cfn_stack["stack_name"])
+
+        experiment_name = rand_name("experiment-")
+        experiment_description = rand_name("description-")
+        experiment = kfp_client.create_experiment(
+            experiment_name,
+            description=experiment_description,
+            namespace=DEFAULT_USER_NAMESPACE,
+        )
+
+        pipeline_id = kfp_client.get_pipeline_id(PIPELINE_NAME)
+        job_name = rand_name("run-")
+
+        run = kfp_client.run_pipeline(
+            experiment.id, job_name=job_name, pipeline_id=pipeline_id
+        )
+
+        assert run.name == job_name
+        assert run.pipeline_spec.pipeline_id == pipeline_id
+        assert run.status == None
```

Comment:
why? this is some temporary state right?

Reply:
yeah i guess this assertion doesn't add any value, will remove